### ISSUE TITLE
Not possible to add custom labels to images using TalkBack

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#18759]
Using talkback it is possible to apply labels to unlabeled images.  Currently an unlabeled image cannot be labeled with a Flutter app.  

See https://support.google.com/accessibility/android/answer/6007066?hl=en

**Comment by jonahwilliams** [comment]
This will not be possible to do unless we want to provide some sort of stable identifier in the semantics node, for example the url for a network image.

**Comment by goderbauer** [comment]
This issue is about a talkback user being unable to add a custom label to an unlabeled image, right? As a developer, you can totally label an unlabeled image by wrapping it in a Semantics widget.

**Comment by jonahwilliams** [comment]
Yes - since we don't provide stable resource ids, navigating to a new page and back will cause the custom label to be lost.
